### PR TITLE
Repositories Claim and ClaimReview

### PR DESCRIPTION
**Pull request by DuardoN**

### CODE REVIEW

File: packages/server/api/controller/claimController.js [hunk 1]
```diff
@@ -1,55 +1,47 @@
...
-        _id: req.params.id
-    }, (err, claim) => {
-        if (err) { res.send(err); }
-        res.json({ message: 'claim successfully deleted' });
-    });
+const ClaimRespository = require('../repository/claim');
+
+module.exports = class ClaimController {
+    listAll() {
+        return new Promise((resolve, reject) => {
```
Comment (thesocialdev) [inline packages/server/api/controller/claimController.js]: Let's use async wait everywhere to keep consistency, what do you think?

File: packages/server/api/repository/claimReview.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        return ClaimReview.find({}).lean();
+    }
+
+    static create(claimReview) {
+        const newClaimReview = new ClaimReview(claimReview);
+        return newClaimReview.save();
+    }
+
+    static getById(claimReviewId) {
+        return ClaimReview.findById(claimReviewId);
```
Comment (thesocialdev) [inline packages/server/api/repository/claimReview.js]: Not populating with claims

File: packages/server/routes/claimReview.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 'use strict';
 
-const claimReview = require('../api/controller/claimReviewController');
+const ClaimReview = require('../api/controller/claimReviewController');
```
Comment (thesocialdev) [inline packages/server/routes/claimReview.js]: For consistency sake, rename ClaimReview to ClaimReviewController. Apply this everywhere for other classes.

Comment (DuardoN) [reply packages/server/routes/claimReview.js]: ok.

File: packages/server/api/controller/claimController.js (hunk 1 above)
Comment (DuardoN) [reply packages/server/api/controller/claimController.js]: It's ok. No problem

File: packages/server/api/repository/claimReview.js (hunk 2 above)
Comment (DuardoN) [reply packages/server/api/repository/claimReview.js]: I forgot populate data in repository.

**Pull request merged**